### ISSUE TITLE
Type not found: h3d.impl.Serializable

### ISSUE THREAD

**Issue by Sean-Clarke** [HeapsIO/heaps#365]
When attempting to run "haxe sao_js.hxml" in the build/sao directory and "haxe base3D_js.hxml" in the build base3D.hxml directory I am met with ../../SampleApp.hx:15 characters 19-56 : Type not found : h3d.impl.Serializable and no .js files having compiled, thus no working sample when the corresponding index.html files are opened. Is there an easy solution to my problem. I have latest Haxe and heaps as of 2018/03/26, and am running on Windows.

Cheers

**Comment by ncannasse** [comment]
Try to remove / update hxbit library

Le mar. 27 mars 2018 04:42, Sean A Clarke <notifications@github.com> a
écrit :

**Comment by Sean-Clarke** [comment]
My hxbit library is version is 1.4.0, which is what it was when I had these issues. I have now tried removing my hxbit library and attempted to run "haxe sao_js.hxml" in the build/sao dir and was met with: Error: Error: Library hxbit is not installed : run 'haxelib install hxbit'. Doing so has renewed my hxbit library once again version 1.4.0, and the above command is once again met with: ../../SampleApp.hx:15 characters 19-56 : Type not found : h3d.impl.Serializable. Although there is clearly something not working, I have moved on from attempting to use these samples, and do not mind if you don't have time to address the issue. However, if you want to figure out what is wrong, I am open to continue troubleshooting.

Cheers
